packages/frontend/src/components/UserCompletionModal/index.tsx

```diff
@@ -15,20 +15,29 @@ import {
     UserCompletionModalCard,
 } from './UserCompletionModal.styles';
 
-const jobTitles = [
+function shuffleArray<T extends unknown>(arr: T[]) {
+    return arr
+        .map((a) => [Math.random(), a] as [number, T])
```

Comment:
Nice approach, I find this more readable, with objects instead of array. What do you think ? 
```

.map(value => ({ value, sort: Math.random() }))
        .sort((a, b) => a.sort - b.sort)
        .map(({ value }) => value)